30_regexp_lab/relab.txt

```diff
@@ -1,17 +1,18 @@
 Use grep to print only the lines from "input" for which:
 
 1. There is a binary number in the line
-
+grep -E [10] input.txt
 2. There is an even binary number in the line
-
+grep -E '0$' input.txt
```

Comment:
same as above, let's use ```\b```:
```  grep -E "\b[10]+0\b"```
this will work most of the time, except for the number 0, because it will be matched by the ```[10]``` part and ```0``` will not be matched.

we can fix it by using an "or" represented by the pipe symbol:
 ```(A|B) # match either A or B```
like this:
```grep -E "\b([10]+0|0)\b"```